test/test_utils.h

```diff
@@ -46,6 +46,19 @@
 namespace celerity {
 namespace detail {
 
+	struct runtime_testspy {
+		static scheduler& get_schdlr(runtime& rt) { return *rt.m_schdlr; }
+		static executor& get_exec(runtime& rt) { return *rt.m_exec; }
+		static size_t get_command_count(runtime& rt) { return rt.m_cdag->command_count(); }
+		static command_graph& get_cdag(runtime& rt) { return *rt.m_cdag; }
+		static std::string print_graph(runtime& rt) {
+			if(auto graph =
+			        rt.m_cdag.get()->print_graph(rt.m_cfg.get()->get_graph_print_max_verts(), *rt.m_task_mngr, *rt.m_reduction_mngr, rt.m_buffer_mngr.get()))
+				return graph.value();
+			return {};
```

Comment:
```suggestion
			if(auto graph =
			        rt.m_cdag.get()->print_graph(rt.m_cfg.get()->get_graph_print_max_verts(), *rt.m_task_mngr, *rt.m_reduction_mngr, rt.m_buffer_mngr.get())) {
				return graph.value();
			}
			return {};
```

I wonder why clang-tidy didn't catch this 🤔.

Reply:
Alternatively you could just `return graph.value_or("");`.